StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentViewController.swift

```diff
@@ -67,6 +67,10 @@ class ConsentViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
@mats-stripe 

Do you have to do this anymore? (for the selector apI)

https://developer.apple.com/documentation/foundation/nsnotificationcenter/1415360-addobserver

> If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.

Reply:
TIL! Thanks for pointing this out, will remove this!